compiler/src/iree/compiler/Codegen/LLVMGPU/Passes.cpp

```diff
@@ -719,32 +719,35 @@ void addGPUBaseLoweringPassPipeline(OpPassManager &pm) {
 }
 
 void addGPUImplicitGEMMPassPipeline(OpPassManager &pm) {
-  auto &nestedModulePM = pm.nest<ModuleOp>();
-  nestedModulePM.addNestedPass<func::FuncOp>(createLLVMGPUIm2ColPass());
-  nestedModulePM.addNestedPass<func::FuncOp>(createTileUsingForallPass(0));
-  nestedModulePM.addPass(createCanonicalizerPass());
-  nestedModulePM.addPass(createCSEPass());
-  nestedModulePM.addNestedPass<func::FuncOp>(
-      createLLVMGPUTileMatmulAndFuseImg2ColPass(1));
```

Comment:
I think we still need this pass to tile reduction dim?

Reply:
Ah, that we do need. Ill add it. Good catch!

Reply:
Cool cool, I will double check the IR tomorrow.